dmaws/deploy.py

```diff
@@ -65,6 +65,19 @@ def create_version(self, version_label, with_sha=False, description='',
 
         return version_label, bool(created_version)
 
+    def prune_old_versions(self, number_to_keep):
+        assert isinstance(number_to_keep, (int, float))
+
+        versions = self.beanstalk.list_application_versions(self.eb_application)
```

Comment:
It's returned with the most recent first and this will remove the tail of that list.